numba/targets/arraymath.py

```diff
@@ -530,46 +581,83 @@ def _trivial_isnan(x):
             return False
         return _trivial_isnan
 
+@register_jitable
+def less_than(a, b):
+    return a < b
 
-@overload(np.nanmin)
-def np_nanmin(a):
-    if not isinstance(a, types.Array):
-        return
-    isnan = get_isnan(a.dtype)
+@register_jitable
+def greater_than(a, b):
+    return a > b
 
-    def nanmin_impl(a):
-        if a.size == 0:
-            raise ValueError("nanmin(): empty array")
-        for view in np.nditer(a):
-            minval = view.item()
-            break
-        for view in np.nditer(a):
-            v = view.item()
-            if not minval < v and not isnan(v):
-                minval = v
-        return minval
+@register_jitable
```

Comment:
Note to self. This is just the sort of place where `@inlinable` would be useful. This doesn't need to be compiled with reliance on the LLVM inliner to inline the function body, the IR just needs inlining at the call site.